### DISCUSSION TITLE
Infinite Loading

### DISCUSSION THREAD

**Discussion by ArnavAhuja1000** [intelowlproject/IntelOwl#1985] [Q&A]
After I log in, the application sometimes gets stuck loading forever. I haven't given it any data and I just sit on the dashboard and switch between the tabs (6h, 24h, etc.) and it seems to be stuck loading forever after the first switch. Is this an issue I can somehow resolve? I am trying to scale this to be used across many devices, but the connection has always seemed finicky.

**Comment by mlodic** [comment]
hey, if that is the first time that you run the application, that is normal and is also[ documented](https://intelowl.readthedocs.io/en/latest/Installation.html#tl-dr). 

To help you to debug your case, we would need to have your [application logs](https://intelowl.readthedocs.io/en/latest/Contribute.html#debug-application-problems). Otherwise it is impossible for us to help you in any way.
